<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditView</class>
 <widget class="QDialog" name="CreditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="type_credit">
     <item>
      <widget class="QRadioButton" name="type_dif">
       <property name="text">
        <string>Дифференцированный</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="type_ayt">
       <property name="text">
        <string>Аннуитетный</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="3" column="0">
      <widget class="QLineEdit" name="input_credit_body"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="input_credit_term"/>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="text_persent">
       <property name="text">
        <string>Процентная ставка, %</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="text_credit_body">
       <property name="text">
        <string>Сумма кредита, руб</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="text_credit_term">
       <property name="text">
        <string>Срок, месяц</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLineEdit" name="input_percent"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="solution">
       <property name="text">
        <string>Рассчитать</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="text_out_term">
     <property name="text">
      <string>Ежемесячный платеж, руб</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="out_term_list"/>
   </item>
   <item>
    <layout class="QGridLayout" name="resalt">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <property name="spacing">
      <number>10</number>
     </property>
     <item row="0" column="2">
      <widget class="QLabel" name="text_2">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="text">
       <property name="text">
        <string>Переплата по кредиту</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="out_overpayment">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="total_2">
       <property name="text">
        <string>Выплата за весь период</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="out_total_playment">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="text_3">
       <property name="text">
        <string>руб.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
